spec/rubocop/cop/rspec/before_after_all_spec.rb

```diff
@@ -0,0 +1,63 @@
+# encoding: utf-8
+
+describe RuboCop::Cop::RSpec::BeforeAfterAll do
+  subject(:cop) { described_class.new }
+
+  context 'when offenses detected' do
+    let(:code) do
+      [
+        'describe MyClass do',
+        '  before(:all) { do_something}',
+        '  after(:all) { do_something_else }',
+        'end'
+      ]
+    end
+
+    let(:expected_error) do
+      'Avoid the use of before/after(:all) '\
+        'as they are not rolled back '\
+        'and may lead to database state leaking between examples'
+    end
+
+    it 'reports 2 offenses' do
```

Comment:
Thoughts on removing this test? The test below (`"reports the lines for these offenses "`) is asserting that there are offenses on lines 2 and 3. Unless I'm missing something that should be equivalent to asserting the number of cops